pkg/security/admission/admission.go

```diff
@@ -205,8 +226,11 @@ func (c *constraint) createProvidersFromConstraints(ns string, sccs []*kapi.Secu
 		var err error
 		resolveUIDRange := requiresPreAllocatedUIDRange(constraint)
 		resolveSELinuxLevel := requiresPreAllocatedSELinuxLevel(constraint)
+		resolveFSGroup := requiresPreallocatedFSGroup(constraint)
+		resolveSupplementalGroups := requiresPreallocatedSupplementalGroups(constraint)
+		requiresNamespaceAllocations := resolveUIDRange || resolveSELinuxLevel || resolveFSGroup || resolveSupplementalGroups
 
-		if resolveUIDRange || resolveSELinuxLevel {
+		if requiresNamespaceAllocations {
 			var min, max *int64
```

Comment:
seems like these should be scoped to the blocks where you need them, so that they never escape.